storybook/cypress/e2e/image-overlay.cy.ts

```diff
@@ -51,6 +51,9 @@ describe("Image Overlay", () => {
 
   it("should close overlay on click outside", () => {
     openOverlay();
+
+    cy.get("dso-image-overlay.hydrated").shadow().matchImageSnapshot();
```

Comment:
Ik vind het geen probleem dat het screenshot moment is verplaatst, maar waarom heb je dit gedaan?

En waarom maken we hier 'n screenshot van de shadow root?

Reply:
Het leek mij wel opportuun om ook de vergroting in beeld te hebben.

![Image Overlay -- should close overlay on click outside snap](https://github.com/dso-toolkit/dso-toolkit/assets/4646174/fbbe5f6f-aea0-419b-99fb-6b736791ae1f)

Reply:
Eric gaat testen of screenshot op hele pagina toereikend is.